src/sentry/conf/server.py

```diff
@@ -1008,6 +1008,8 @@ def create_partitioned_queues(name):
     "organizations:dashboards-edit": True,
     # Enable dashboard widget library
     "organizations:widget-library": False,
+    # Enable issue widgets in dashboards
+    "organizations:issues-in-dashboards": True,
```

Comment:
Should this be false by default?

Reply:
🤦 yep should be default to false, updated!